code/game/machinery/computer/card.dm

```diff
@@ -49,8 +49,9 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		/datum/job/blueshield,
 		/datum/job/nanotrasenrep,
 		/datum/job/chaplain,
-		/datum/job/officer
-	)
+		/datum/job/officer,
+		/datum/job/qm)
```

Comment:
```suggestion
		/datum/job/qm
)
```

Reply:
I hate that this is how we do it. It looks so messy.